examples/helpers/cells.js

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
I moved all the cells here to help clean things up.  I also switched them all to PureComponents instead of stateless functions to encourage practices for best performance.